gammagl/utils/split.py

```diff
@@ -0,0 +1,25 @@
+import tensorlayerx as tlx
+import numpy as np
+
+def split(y):
```

Comment:
This util is useless, remove it.

Reply:
`from sklearn.model_selection import train_test_split`